speakeasy/windows/common.py

```diff
@@ -510,30 +515,46 @@ def get_exports_size(self, name, exports):
             exp_size += (0x4 + 0x4 + 0x4)
 
         return exp_size
+    
+    def align_file(self):
```

Comment:
```suggestion
    def pad_file(self):
```

perhaps?

i think aligning would be changing the base address, not adding to the end